docs/tutorial.rst

```diff
@@ -108,139 +112,278 @@ For reproducibility we can also choose set seed for the simulator using the
+
+First, let's use the ``--help-models`` option to see the selection of demographic
+models available to us:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This prints detailed information about all of the available models to
+the terminal.
+In this tutorial, we will use the model of African-American admixture from
+`2011 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.  
+From the help output (or the :ref:`Catalog <sec_catalog>`),
+we can see that this model has id ``america``,
+and allows samples to be drawn from 4 contemporary populations representing African,
+European, Asian and African-American groups.
+
+Using the ``--help-genetic-maps`` option, we can also see what recombination maps
+are available:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-genetic-maps
+
+Let's go with ``HapmapII_GRCh37``.
+The next command simulates 4 samples of chromosome 1 from each of the four
+populations, and saves the output to a file called ``afr-america-chr1.trees``.
+For the purposes of this tutorial, we'll also specify a random seed using the
+``-s`` option.
+(Note: This took around 8 minutes to run on a laptop.)
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr1 -o afr-america-chr1.trees -s 13 -g HapmapII_GRCh37\
+    --model america 4 4 4 4
+
+We should now have a file called ``afr-america-chr1.trees``.
+Our work with ``stdpopsim`` is done; we'll now switch to a Python console and import
+the ``tskit`` package to load and analyse this simulated tree sequence file.
 
 .. code-block:: python
 
-    species = stdpopsim.get_species("homsap")
-    contig = species.get_contig("chr22", length_multiplier=0.1)
-    model = stdpopsim.PiecewiseConstantSize(species.population_size)
-    samples = model.get_samples(10)
-    engine = stdpopsim.get_default_engine()
-    ts = engine.simulate(model, contig, samples)
+    >>> import tskit
+    >>> ts = tskit.load("afr-america-chr1.trees")
+
+Recall that `genetic divergence` is the probability that two randomly sampled
+chromosomes differ at a nucleotide base.
+For a given pair of populations, a pair-specific divergence value is obtained
+by randomly sampling one chromosome from each population.
+These quantities can be estimated directly from our sample using tskit's
+inbuilt :meth:`tskit.TreeSequence.diversity` method.
+
+By looking at
+`the documentation <https://tskit.readthedocs.io/en/latest/python-api.html#tskit.TreeSequence.divergence>`_
+for this method, we can see that we'll need two inputs: ``sample_sets`` and
+``indexes``.
+Let's think about what these inputs are, and how we can obtain them with 
+Python commands.
+In our case, the sample sets correspond to the lists 
+of sample chromosomes (nodes) from each separate population.
+We can obtain the necessary list of lists like this:
+
+.. code-block:: python
 
+    >>> sample_list = []
+    >>> for pop in range(0, ts.num_populations):
+    ...     sample_list.append(ts.samples(pop).tolist())
```

Comment:
you don't need the `.tolist()` here (but maybe it makes it print out nicer on the next line?)

Reply:
The tskit documentation for this functions specifies a 'list of lists of node IDs', so I just thought it would be most transparent if this is exactly what we calculated here

Reply:
sure, why not